tests/js/all-tests.js

```diff
@@ -1,4 +1,9 @@
 // Run all tests in this package in series.
 "use strict";
+
+var events = require("events");
+events.EventEmitter.defaultMaxListeners = 100;
```

Comment:
We could not find a root cause in a brief exploration.  We have documented this and left it in place for now.